packages/client/src/proxy.js

```diff
@@ -4,10 +4,28 @@ import http from 'http';
 import https from 'https';
 import logger from '@percy/logger';
 import { stripQuotesAndSpaces } from '@percy/env/utils';
+import { PacProxyAgent } from 'pac-proxy-agent';
```

Comment:
missed adding this dependency ?

Reply:
yes, added now. Please check that it is at the right place. (It is in the base package json)